security/practical-security.md

```diff
@@ -0,0 +1,10 @@
+At one time, not so long ago, I used the same weak password for all of my sites that I visited. My computer was only protected by the basic security software that came with windows. However as the world of malicious hacking and my own knowledge of this hacking increased so did my protection increased.
+
+Now I use a password manager that I use to help me create strong passwords. All super sensative information like my bank account and medical records a 2-factor authenicated. To protect it even farther I only log into those accounts using a a linux live key that saves no infomation. My computer has virus protection and a firewall. My computer entry is my fingerprint. I very seldom download anything when. When I do, I download in a virtual drive. I also check the sha on the software. I never open email's attachments outside of my key drive. Because I am a programmer, I add little more protection to myself. I use ssh keys for my git transactions
+
+That being said, there are some things I could improve on: Changing my passwords more frequently and use passphrases instead of passwords, use gpg for github, use keybase and learn more about Kali, Docker and sandboxing.
```

Comment:
Signing your commits is definitely a good way to protect yourself and your code. Keybase is also a wonderful tool that I believe all programmers should use.